src/waveform/renderers/waveformrenderbeat.cpp

```diff
@@ -49,25 +67,31 @@ void WaveformRenderBeat::draw(QPainter* painter, QPaintEvent* /*event*/) {
     const double lastDisplayedPosition =
             m_waveformRenderer->getLastDisplayedPosition();
 
-    // qDebug() << "trackSamples" << trackSamples
-    //          << "firstDisplayedPosition" << firstDisplayedPosition
-    //          << "lastDisplayedPosition" << lastDisplayedPosition;
+    // Calculate beat length
+    double beatLength = (c_secondsPerMinute * trackInfo->getSampleRate() / trackInfo->getBpm()) * c_numberOfChannels;
 
+	// Get the visible beats
```

Comment:
I'm using Eclipse. I already had the preferences set to use 4 spaces for indenting instead of tabs. I have added the preference to see white spaces, so I must be able to see if the editor is not doing it as expected. Sorry for the inconvenient.

I'm supposed to "destroy" tabs if I do find them in the code I'm modifying? In principle I follow the philosophy of minimum changes.

Reply:
Yes, if you come across code that uses tabs to indent, please replace those with 4 spaces.